telnyx/api_resources/phone_number.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import, division, print_function
+
+from telnyx import error, six, util
+from telnyx.api_resources.abstract import (
+    DeletableAPIResource,
+    ListableAPIResource,
+    UpdateableAPIResource,
+    nested_resource_class_methods,
+)
+from telnyx.six.moves.urllib.parse import quote_plus
+
+
+@nested_resource_class_methods("voice", operations=["list"], pluralize_path=False)
+@nested_resource_class_methods("voice", operations=["update"], pluralize_path=False)
```

Comment:
If `operations` is a list, can you combine them like
```suggestion
@nested_resource_class_methods("voice", operations=["list", "update"], pluralize_path=False)
```
?

Reply:
Good catch. I originally listed these out in order of the API documentation so I would be sure I didn't miss an endpoint. Forgot to clean it up.